### ISSUE TITLE
Posibilidad de ayudarles con el desarrollo

### ISSUE THREAD

**Issue by Bryan-Herrera-DEV** [redmujeresmatematicasECU/red-mujeres-matematicas-ECU#7]
Con el fin de impulsar la web quisiera ayudarles sin ningún animo de lucro a tener una mejor presencia digital.

Soy Bryan, tengo 20 años de edad y llevo desarrollando mas de 6 años.

**Comment by redmujeresmatematicasECU** [comment]
Hola, claro muchas gracias por tu intención de ayudar a nuestra red. Yo trabajo también en react aunque tengo 3 años de experiencia y si mucho tiempo como para trabajar directamente en este proyecto, lo que te podría sugerir si estas de acuerdo es trabajar entre los dos.

**Comment by Bryan-Herrera-DEV** [comment]
Claro, me parece perfecto, te puedo proporcionar mi LinkedIn y trabajamos en la implementacion de esta propuesta.

https://www.linkedin.com/in/bryan-herrera-dev/

**Comment by redmujeresmatematicasECU** [comment]
Hola Bryan, disculpa el delay de comunicacion tan grande. Entre el trabajo y otras tareas personales reciene stoy teniendo tiempo para poder trabajar en la pagina de la red. 
Lo que te propongo, si te conviene claro esta podemos organizar una mini reunion para ver lo que se debe mejorar en un principio y empezar a hacerlo y tmb para hablar sobre lo que se debe sacar de aqui mas o menos en unos 2 meses de ser posible. 
Dado el cambio de horario creo que seria mejor tener esa reunion en fin de semana sino podria ser en tipo la hora de almuerzo para ti y cena para mi. De mi parte estoy disponible los lunes, martes despues de las 18h (hora Francia) los miercoles y jueves a partir de las 19h30 y bueno en sabado y domingo seria igual en la tarde noche de aqui.
Me avisas tus disponibilidades para poder organizarnos.
